apps/family-public-law/fpl-case-service/fpl-case-service.yaml

```diff
@@ -8,12 +8,19 @@ spec:
   releaseName: fpl-case-service
   values:
     java:
-      replicas: 2
+      replicas: 4
```

Comment:
Why has the number replicas been increased from 2 to 4? I'd have thought this should stay at 2...